AwaMelvine/fast-food-fast
# Feat crud menu items 161218536

#### What does this PR do?
Implements create, read, update and delete food items on the menu using the client application. 

#### Description of Task to be completed?
- Implement code to create food item
- Implement code to read the whole menu as well as a specific food item by ID
- Implement code to modify and update a particular food item
- Implement code to delete a particular food item

#### How should this be manually tested?
1. For Create and Read actions, visit the create items page on the admin section and correctly fill out the form with details about a new item. Then click on the save item button. This should take you to the admin menu page where all available items are listed including the one you just created.
2. For Delete action, click on the delete button of a particular food item and the item will be deleted and will no longer appear in the menu with other items.
3. For Update action, click on the edit button of a particular item. On the Edit Item form, modify the item and click the Update Food Item button. This should take you to the admin menu page displaying all food items including the recently modified food item.

#### What are the relevant pivotal tracker stories?
[#161218536] Admin user should be able to create and read menu items on the frontend
[#161547702] Admin user should be able to delete food item from the client
[#161258370] Admin user should be able to edit and update an item on the client
